Installation of Zulu 64 bit fails (at least on aarch64)

Installing the 64bit Zulu Java fails on aarch64.

To reproduce on aarch64:

1. Run openhabian-config
2. Select Fresh/Manual Install
3. Select only Zulu Java 64 bit
4. Start installation process

Zulu 32bit for armhf is installed instead of the 64 bit version.

The reason is that the java_arch variable is not written correctly into /etc/openhabian.config.

functions/config.bash saves an option called "option" into openhabian.config:
```
## Update java architecture in config file
## Valid options: "32-bit" & "64-bit"
update_config_java() {
  cond_redirect grep -q '^java_arch' "$CONFIGFILE" && sed -i "s/^java_arch.*/java_arch=$1/" "$CONFIGFILE" || echo "option=$1" >> "$CONFIGFILE"
  # shellcheck disable=SC1090
  source "$CONFIGFILE"
}
```

The line
```
   cond_redirect grep -q '^java_arch' "$CONFIGFILE" && sed -i "s/^java_arch.*/java_arch=$1/" "$CONFIGFILE" || echo "option=$1" >> "$CONFIGFILE"
```

Should be
```
  cond_redirect grep -q '^java_arch' "$CONFIGFILE" && sed -i "s/^java_arch.*/java_arch=$1/" "$CONFIGFILE" || echo "java_arch=$1" >> "$CONFIGFILE"
```

This fix works for me. :)

--- a/functions/system.bash
+++ b/functions/system.bash
@@ -9,17 +9,7 @@ whiptail_check() {
 
 system_upgrade() {
   echo -n "$(timestamp) [openHABian] Updating repositories and upgrading installed packages... "
-  # TODO: Remove updating the key on each update after some time (in 2020)
-  cond_redirect wget -O openhab-key.asc 'https://bintray.com/user/downloadSubjectPublicKey?username=openhab'
-  if ! cond_redirect apt-key add openhab-key.asc; then
-    echo "FAILED (updating OH key)"
-    echo -n "attempting to continue..."
-  else
-    rm -f openhab-key.asc
-  fi
   cond_redirect apt-get --yes upgrade
-  # shellcheck disable=SC2154
-  if cond_redirect java_install_or_update "$java_arch"; then echo "OK"; else echo "FAILED"; exit 1; fi
 }
 
 basic_packages() {

